gemfiles/activesupport3.2.gemfile

```diff
@@ -3,6 +3,7 @@
 source "https://rubygems.org"
 
 gem "activesupport", "~> 3.2.0"
+gem "memcache-client"
```

Comment:
I've addressed that. Didn't know about `appraisal`, very interesting stuff!
I've also refactored the `Rack::Attack::StoreProxy.build` method, which was getting too cluttered in my opinion (but I could move that to a separate PR if you think it's necessary).